docs/src/user_docs/assembly/code_organization.md

```diff
@@ -3,16 +3,16 @@ A Miden assembly program is just a sequence of instructions each describing a sp
 
 In turn, Miden assembly instructions are just keywords which can be parameterized by zero or more parameters. The notation for specifying parameters is *keyword.param1.param2* - i.e., the parameters are separated by periods. For example, `push.123` instruction denotes a `push` operation which is parameterized by value `123`.
 
-Currently, Miden assembly provides two types of code organization blocks: *scripts* and *procedures*. In the future, additional blocks such as modules and functions will be added.
+Currently, Miden assembly provides two types of code organization blocks: *programs* and *procedures*. In the future, additional blocks such as modules and functions will be added.
 
-### Scripts
-A *script* block is used to define an executable program. A script must start with a `begin` instruction and terminate with an `end` instruction. For example:
+### Programs
+A *program* block is used to define an executable program. A program must start with a `begin` instruction and terminate with an `end` instruction. For example:
 ```
 begin
     <instructions>
 end
 ```
-When Miden assembly code is executed, the execution starts at the first instruction following the `begin` instruction of the script. A script block is expected to be the last block in a program and can be followed only by comment blocks.
+When Miden assembly code is executed, the execution starts at the first instruction following the `begin` instruction of the program. A program block is expected to be the last block in a program and can be followed only by comment blocks.
```

Comment:
I think the last sentence here is a bit confusing now and should maybe be rephrased.